File: pinyougou-web-shop/src/main/webapp/js/service/seckillService.js
app.service("seckillService",function($http){
    this.findMarket = function () {
        return $http.get('www.baidu.com')
    }

    this.search = function (page,rows) {
        return $http.get('../seckill/findAllBySeller.do')
    }

    this.findOne = function (id) {
        return $http.get('../seckill/findOne.do?id='+id);
    }

    this.commitSeckill = function (seckillGoods) {
        return $http.post('../seckill/commitSeckill.do',seckillGoods )
    }
})

File: pinyougou-interface/src/main/java/cn/itcast/core/service/seckill/SeckillService.java
package cn.itcast.core.service.seckill;

import cn.itcast.core.entity.PageResult;
import cn.itcast.core.pojo.item.Item;
import cn.itcast.core.pojo.seckill.SeckillGoods;

import java.util.List;

/**
 * @Description:
 * @Author: zc
 * @CreateDate: 2019/2/14$ 13:38$
 */
public interface SeckillService {
    List<Item> findAllSKUBySeller(String sellerId);

    Item findOne(Long id);

    void commitSeckill(SeckillGoods sellerId);
}
